app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="A user-friendly tool to ease the distribution of very large files"
+DESCRIPTION="jigit aims to be a user-friendly wrapper for mkimage. mkimage is \
+a simple but fast tool to reconstruct large image files from .jigdo and .template \
+files. It doesn't have any logic to cope with downloading missing files, but will \
+list the missing files that are needed. It is also convenient in case the user has \
```

Comment:
Hmm, I just realized that my suggestion for `DESCRIPTION` needs a little fix because the mkimage tool is called jigit-mkimage.
```diff
-DESCRIPTION="jigit aims to be a user-friendly wrapper for mkimage. mkimage is \
-a simple but fast tool to reconstruct large image files from .jigdo and .template \
-files. It doesn't have any logic to cope with downloading missing files, but will \
-list the missing files that are needed. It is also convenient in case the user has \
-a partially outdated optical media or ISO image containing at least some files \
-which are not outdated."
+DESCRIPTION="jigit aims to be a user-friendly wrapper for jigit-mkimage. \
+jigit-mkimage is a simple but fast tool to reconstruct large image files from \
+\".jigdo\" and \".template\" files. \
+It doesn't have any logic to cope with downloading missing files, but will \
+list the missing files that are needed. \
+It is also convenient in case the user has a partially outdated optical media \
+or ISO image containing at least some files which are not outdated."
```